src/app/pages/device-options/device-options.component.ts

```diff
@@ -266,7 +269,18 @@ export class DeviceOptionsComponent implements OnInit, OnDestroy {
   onLevelSet(device: Switch) {
     this.service
       .setDimLevel(device.idx, device.Level)
-      .pipe(take(1), takeUntil(this.unsubscribe$))
+      .pipe(
+        tap((resp: DomoticzResponse<Switch>) => {
+          console.log("↩️ Response from Server when set level-->" + JSON.stringify(resp, null, 4));
```

Comment:
Eviter console log sur les requêtes

Reply:
Keep this console just for debugging in detail. Same reason like someone above.